is_core/forms/fields.py

```diff
@@ -47,14 +47,14 @@ class ButtonField(ReadonlyField):
     def __init__(self, label, attrs=None, widget=None):
         attrs = attrs or {}
         widget = widget or self.widget
-        super(ButtonField, self).__init__(required=False, label='', initial=label, widget=widget(attrs=attrs))
+        super().__init__(required=False, label='', initial=label, widget=widget(attrs=attrs))
```

Comment:
to initial se nastavuje na label schválně?

Reply:
ano, v buttonu chceš mít ve value label (aby byl správně popsaný)